roles/pulibrary.redis/templates/redis_ulimit.conf.j2

```diff
@@ -0,0 +1,3 @@
+# {{ ansible_managed | comment }}
+redis soft nofile 65535
+redis hard nofile 65535
```

Comment:
should the soft nofile be a different number?

Reply:
If this was your desktop machine you'd want "the warning" that soft limit would buy us. If redis is consuming all of those resources we want to know with a hard crash.